README.md

```diff
@@ -25,3 +25,18 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+//////
+T(n) = [(1, if n =<1) (3T(n/3)+n, if n >= 1)]
```

Comment:
Recurrence relation incorrect -- how much work are you doing to combine the partial results?